asset_dashboard/static/js/PortfolioPlanner.js

```diff
@@ -513,14 +534,26 @@ class PortfolioPlanner extends React.Component {
     }
   }
 
+  filterPhase(project) {
+    if (this.state.selectedPhase) {
+      // Ensure we're matching on the phase value displayed on the table
+      const selectedOption = this.props.phaseOptions.filter(
+        (source) => source['value'] == this.state.selectedPhase
+      )[0]
+      return project.phase == selectedOption.label
```

Comment:
Decided to not use the `within()` helper function here (and by extension in for the funding sources) so that we could get an exact match and not have `Construction` match with both `Construction` and `Construction Engineering`.